controllers/watermarkpodautoscaler_controller.go

```diff
@@ -252,7 +252,7 @@ func (r *WatermarkPodAutoscalerReconciler) reconcileWPA(ctx context.Context, log
 	if rescale {
 		setCondition(wpa, autoscalingv2.AbleToScale, corev1.ConditionTrue, datadoghqv1alpha1.ConditionReasonReadyForScale, "the last scaling time was sufficiently old as to warrant a new scale")
 		if wpa.Spec.DryRun {
-			logger.Info("DryRun mode: scaling change was inhibited", "currentReplicas", currentReplicas, "desiredReplicas", desiredReplicas)
+			logger.Info("DryRun mode: scaling change was inhibited", wpa.GetLogAttrs("currentReplicas", currentReplicas, "desiredReplicas", desiredReplicas))
```

Comment:
instead of updating every `logger.Info()` call.
you can add at the beginning of the function `Reconcile()` [here](https://github.com/DataDog/watermarkpodautoscaler/pull/113/files#diff-70d5250d0b22ca57247df02274172975492cd9862cc800a40ec91644a462d9ddR109)

the logs attributes
```golang
log = log.WithValues(GetLogAttrsFromWPA(instance))
```
this values will be added to any log lines.